<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MidiMapDialog</class>
 <widget class="QDialog" name="MidiMapDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>487</width>
    <height>270</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QTabWidget" name="mapperTabBox">
     <property name="currentIndex">
      <number>0</number>
     </property>
     <widget class="QWidget" name="MidiInputTab">
      <attribute name="title">
       <string>Midi Input</string>
      </attribute>
      <layout class="QGridLayout" name="gridLayout">
       <item row="0" column="0">
        <widget class="QLabel" name="label">
         <property name="text">
          <string>Channel</string>
         </property>
        </widget>
       </item>
       <item row="0" column="1">
        <widget class="QLineEdit" name="inChannelEdit">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="2" column="0">
        <widget class="QLabel" name="label_2">
         <property name="text">
          <string>Status</string>
         </property>
        </widget>
       </item>
       <item row="4" column="0">
        <widget class="QLabel" name="label_3">
         <property name="text">
          <string>Data 1</string>
         </property>
        </widget>
       </item>
       <item row="5" column="0">
        <widget class="QLabel" name="label_4">
         <property name="text">
          <string>Data 2</string>
         </property>
        </widget>
       </item>
       <item row="2" column="1">
        <widget class="QLineEdit" name="inStatusEdit">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="4" column="1">
        <widget class="QLineEdit" name="inData1Edit">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="5" column="1">
        <widget class="QLineEdit" name="inData2Edit">
         <property name="text">
          <string/>
         </property>
        </widget>
       </item>
       <item row="0" column="2">
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="7" column="1">
        <spacer name="verticalSpacer">
         <property name="orientation">
          <enum>Qt::Vertical</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>20</width>
           <height>40</height>
          </size>
         </property>
        </spacer>
       </item>
       <item row="6" column="1">
        <widget class="QCheckBox" name="midiCaptureCheckBox">
         <property name="text">
          <string>Midi Capture</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
     <widget class="QWidget" name="ActionTab">
      <attribute name="title">
       <string>Action</string>
      </attribute>
      <widget class="QWidget" name="layoutWidget">
       <property name="geometry">
        <rect>
         <x>10</x>
         <y>10</y>
         <width>451</width>
         <height>264</height>
        </rect>
       </property>
       <layout class="QFormLayout" name="formLayout">
        <item row="0" column="0">
         <widget class="QRadioButton" name="outMidiRadioButton">
          <property name="text">
           <string>Midi Message</string>
          </property>
          <property name="checked">
           <bool>true</bool>
          </property>
         </widget>
        </item>
        <item row="0" column="1">
         <widget class="QFrame" name="frame">
          <property name="frameShape">
           <enum>QFrame::StyledPanel</enum>
          </property>
          <property name="frameShadow">
           <enum>QFrame::Raised</enum>
          </property>
          <layout class="QGridLayout" name="gridLayout_2">
           <item row="0" column="0">
            <widget class="QLabel" name="label_8">
             <property name="text">
              <string>Channel</string>
             </property>
            </widget>
           </item>
           <item row="0" column="1">
            <widget class="QLineEdit" name="outChannelEdit">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="1" column="0">
            <widget class="QLabel" name="label_7">
             <property name="text">
              <string>Status</string>
             </property>
            </widget>
           </item>
           <item row="1" column="1">
            <widget class="QLineEdit" name="outStatusEdit">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="2" column="0">
            <widget class="QLabel" name="label_5">
             <property name="text">
              <string>Data 1</string>
             </property>
            </widget>
           </item>
           <item row="2" column="1">
            <widget class="QLineEdit" name="outData1Edit">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
           <item row="3" column="0">
            <widget class="QLabel" name="label_6">
             <property name="text">
              <string>Data 2</string>
             </property>
            </widget>
           </item>
           <item row="0" column="2">
            <widget class="QComboBox" name="outChannelCombo">
             <item>
              <property name="text">
               <string>Add</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Assign</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="1" column="2">
            <widget class="QComboBox" name="outStatusCombo">
             <item>
              <property name="text">
               <string>Add</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Assign</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="2" column="2">
            <widget class="QComboBox" name="outData1Combo">
             <item>
              <property name="text">
               <string>Add</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Assign</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="3" column="2">
            <widget class="QComboBox" name="outData2Combo">
             <item>
              <property name="text">
               <string>Add</string>
              </property>
             </item>
             <item>
              <property name="text">
               <string>Assign</string>
              </property>
             </item>
            </widget>
           </item>
           <item row="3" column="1">
            <widget class="QLineEdit" name="outData2Edit">
             <property name="text">
              <string/>
             </property>
            </widget>
           </item>
          </layout>
         </widget>
        </item>
        <item row="1" column="0">
         <widget class="QRadioButton" name="ouKeyRadioButton">
          <property name="text">
           <string>Key Sequence</string>
          </property>
         </widget>
        </item>
        <item row="1" column="1">
         <widget class="QLineEdit" name="outKeyEdit">
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </widget>
     </widget>
    </widget>
   </item>
   <item>
    <widget class="QDialogButtonBox" name="buttonBox">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox</sender>
   <signal>accepted()</signal>
   <receiver>MidiMapDialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>248</x>
     <y>254</y>
    </hint>
    <hint type="destinationlabel">
     <x>157</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox</sender>
   <signal>rejected()</signal>
   <receiver>MidiMapDialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>316</x>
     <y>260</y>
    </hint>
    <hint type="destinationlabel">
     <x>286</x>
     <y>274</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>midiCaptureCheckBox</sender>
   <signal>clicked()</signal>
   <receiver>MidiMapDialog</receiver>
   <slot>midiCaptureClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>142</x>
     <y>181</y>
    </hint>
    <hint type="destinationlabel">
     <x>184</x>
     <y>131</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>ouKeyRadioButton</sender>
   <signal>clicked()</signal>
   <receiver>MidiMapDialog</receiver>
   <slot>keySequenceClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>79</x>
     <y>203</y>
    </hint>
    <hint type="destinationlabel">
     <x>184</x>
     <y>131</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>outMidiRadioButton</sender>
   <signal>clicked()</signal>
   <receiver>MidiMapDialog</receiver>
   <slot>midiMessageClicked()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>76</x>
     <y>59</y>
    </hint>
    <hint type="destinationlabel">
     <x>184</x>
     <y>131</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>midiCaptureClicked()</slot>
  <slot>midiMessageClicked()</slot>
  <slot>keySequenceClicked()</slot>
 </slots>
</ui>
